HTML + basic JavaScript Capstone 1

khanh-devos opened the pull request:
This project Capstone1 includes 2 pages Home(index.html) and About(about.html), both have mobile view and desktop view.
- the design is flexible with images and text content, so I use some from outside
- Bootstrap5 is also used inside this project.
- javascript is used for listing Feature Speakers and Programs at Home page.

khanh-devos committed:
setup readme file, scss, gitignore, install node, bootstrap

khanh-devos committed:
bild home page

- setup home page is index.html
- install bootstrap locally in order to override it but SCSS
- setup .gitignore file
- reponsive mainly by bootstrap

khanh-devos committed:
add top bar(facebook-twitter bar)

khanh-devos committed:
fix header bar padding

khanh-devos committed:
add About page

khanh-devos committed:
add linters

khanh-devos committed:
add and fix linters

khanh-devos committed:
unlock linter

khanh-devos committed:
fix linter err

khanh-devos committed:
fix height headline

ShadyShawkat commented on README.md:
```diff
@@ -60,57 +26,45 @@ After you're finished please remove all the comments and instructions!
-
 - [Live Demo Link](https://google.com)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
- [x] Please make sure that the live demo is working properly.

ShadyShawkat commented on README.md:
```diff
@@ -237,52 +129,44 @@ Example:
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank Microverse and code reviewers
```
- [x] As per the project requirements, please give appropriate credit to the author of the original design.

ShadyShawkat commented on index.html:
```diff
@@ -0,0 +1,163 @@
+        </h4>
+      </div>
+
+    </section>
```
- Kindly make sure to stick as much as possible to the given template, which includes:
   - [x] Increasing the font size to match the given template.
   - [x] Increasing the spacing to the left of the text.
   - [x] Changing the font color of the text to match the given design.
   - [x] Applying opacity to the background image of this section so that it doesn't oversaturate the section and make the text unreadable.
   
| Your design | Original design |
| --- | --- |
| ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/795b4f5a-5aaf-4748-bbb2-208a7e1be674) | ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/04879b64-7639-4844-8cef-48788c8387d2) |

ShadyShawkat commented on index.html:
```diff
@@ -0,0 +1,163 @@
+      <h2 class="text-white">Main Progarm</h2>
+      <div class="line"></div>
+            
+      <div id="main-container" class="row g-2 row-cols-1 row-cols-md-2 row-cols-lg-3 row-cols-xl-5"></div>
```
- [ ] Kindly introduce a small gap between each item to match the original design.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/1b845565-8a76-4c37-a583-93bf343398b6) | ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/41edd0a5-7033-4988-a277-775ad2c3ef3a) |

ShadyShawkat commented on index.html:
```diff
@@ -0,0 +1,163 @@
+
+    </section>
+
+    <section id="social" class="w-100 bg-dark">
+      <h2>Partner</h2>
+      <div class="line"></div>
+      
+      <div class="d-flex flex-row flex-wrap justify-content-evenly">
+        <a id="soc-a" href="">
+          <span><img src="./imgs/socials/mozilla.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img  src="./imgs/socials/googlekorea.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img src="./imgs/socials/naver.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img src="./imgs/socials/daumkakaologo.svg" alt=""></span></a>
+        <a id="soc-a" href="">
+          <span><img src="./imgs/socials/airbnb.svg" alt=""></span></a>
+      </div>
+
+    </section>
+
+    <section id="contact" class="w-100 text-center">
+      <a id="con-a" href="#">
+        <span> <img src="./imgs/BatteryTech.png" alt="contact">
+        </span>
+        </a> 
+    </section>
```
- [ ] Please be sure to remove the "Partners" and "footer" sections from the home page on the mobile version to match the given design.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/4ab76620-71bf-46a3-9e19-45e200eacdea) | ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/f340bad5-0c77-4903-9aed-968545582199) |

ShadyShawkat commented on about.html:
```diff
@@ -0,0 +1,190 @@
+  </header>
+    
+  <main>
+    <section id="headline" class="w-100 position-relative">
+      <img id="img1" width="100%" height="500px" src="./imgs/about.jpg" alt="home image"
+      >
+      <img id="img2" width="100%" height="500px" src="./imgs/about2.jpg" alt="home image"
+      >
+      <div id="home-content" class="position-absolute top-0 h-100 d-flex flex-column justify-content-center align-items-center">
+        <h3 class="text-white d-md-none">
+            EU Largest Tech Conference:
+        </h3>
+
+        <h1 id="conference-title" class="text-white text-center">
+            Visit The Electric & Hybrid <br> 
+            Vehicle Technology Expo Europe:
+        </h1>
+        
+        <div class="line d-none d-lg-block"></div>
+        
+        <p class="text-white text-center d-none d-lg-block">Explore topics in detail with an interactive learning experience on core subject matters in advanced batteries and electric & hybrid vehicles.
+        Explore topics in detail with an interactive learning experience on core subject matters in advanced batteries and electric & hybrid vehicles. <br>
+        Explore topics in detail with an interactive learning experience on core subject matters in advanced batteries and electric & hybrid vehicles.    
+        </p>    
+        <p class="text-white text-center">Meet battery manufacturers, suppliers, engineers, thought leaders and decision-makers for a conference and battery tech expo focused on the latest developments in the advanced battery and automotive industries.</p>
+
+        <h2 id="head-h2" class="text-white text-center">
+            Please contact us per Email for nay further questions<br>
+        </h2>
+            
+        <a href="" class="text-white">#EVTechExpo
+          <img class="p-1 d-none d-lg-inline-block"
+            src="./imgs/icons/ArrowLeftToRight.svg" alt="arrow left to right" />
+        </a>
+            
+        
+        </h2>
+      </div>
+
+    </section>
```
- Kindly make sure to stick as much as possible to the given template, which includes:
   - [ ] Increasing the spacing to the left and right of the text so it's more centered.
   - [ ] Wrapping the middle text section to be in a white background container, to match the original template.
   - [ ] Changing the font color of the text to match the given design.
   - [ ] Applying opacity to the background image of this section so that it doesn't oversaturate the section and make the text unreadable.
   
| Your design | Original design |
| --- | --- |
| ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/7bbbb170-1902-40fe-b868-97ad3b123ff3) | ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/1a75588d-5a7c-4484-83a6-7e61a5d79a98) |

ShadyShawkat commented on about.html:
```diff
@@ -0,0 +1,190 @@
+      <div id="past-line"></div>
+      <p id="past-p" >have a look at EU's most important event in batteries and EV tech industries.</p>
+
+      <div id="video-container" class="row row-cols-1 row-cols-md-2 justify-content-between">
+        <div class="col">
+          <iframe width="100%" height="100%" src="https://www.youtube.com/embed/WhBTyulrktc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen
+          ></iframe>
+        </div>
+
+        <div class="col">
+          <iframe width="100%" height="100%" src="https://www.youtube.com/embed/L4bwJAt9iC8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+        </div>
+      </div>
```
- [x] I like how you replaced the images with `iframes`, it's a great look. However, for now, please follow the original template. Kindly make sure this section has two images with centered texts in the middle.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/b2f628b1-1668-4a9c-83d2-74697ca42259) | ![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/a0ddfd14-63b6-4b59-9853-1ffeea36e138) |

ShadyShawkat commented on about.html:
```diff
@@ -0,0 +1,190 @@
+
+    </section>
+
+    <section id="contact" class="w-100 text-center">
+      <a id="con-a" href="#">
+        <span> <img src="./imgs/BatteryTech.png" alt="contact">
+        </span>
+        </a> 
+    </section>
```
- [x] Please make sure that the footer section looks appropriate on the desktop view and that the images don't take the full width with a different background. You can use [`filter invert`](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/invert) to achieve so.

![image](https://github.com/khanh-devos/Capstone1-conference/assets/62290545/abe6fff2-24eb-4c6d-8958-125c8856ae69)

ShadyShawkat reviewed (changes requested):
## Hi @khanh-devos, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

![Image](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

khanh-devos committed:
fix comments
- modify reader live demo link
- modify padding
- modify header background opacity and font color, padding
- modify past conference link, remove iframe
- use filter for contact image at About page

levy002 reviewed (approved):
## Hello @khanh-devos 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

  ![unnamed](https://github.com/HERMON-1995/OOP-school-library/assets/87197412/74ff9d90-dd35-49c7-9a85-e33a0ac154b7)


## Highlights
- No linter error :heavy_check_mark: 
- You implemented all suggested changes :heavy_check_mark: 
 
## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure to work on your design between the `768px` and `1021px` screens which must adapt to the desktop version designs but the content is not aligned with the provided design. Kindly adapt your website to those screens to match the design as much as possible.

  - ### YOUR DESIGN
    
    ![chrome-capture-2023-4-25 (1)](https://github.com/khanh-devos/Capstone1-conference/assets/87197412/57361269-d17b-4c96-bd99-dcb7a1681417)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

khanh-devos committed:
modify alignment of both pages as reducing to the point 768px.

khanh-devos committed:
fix linters errors
